sklearn/linear_model/tests/test_logistic.py

```diff
@@ -90,6 +91,18 @@ def test_predict_3_classes():
     check_predictions(LogisticRegression(C=10), X_sp, Y2)
 
 
+def test_lr_liblinear_warning():
+    n_samples, n_features = iris.data.shape
+    target = iris.target_names[iris.target]
+
+    for LR in [LogisticRegression(solver='liblinear', n_jobs=2),
+               LogisticRegressionCV(solver='liblinear', n_jobs=2)]:
```

Comment:
I don't see any early exit for liblinear before [this parallelism](https://github.com/zyxue/scikit-learn/blob/497b59a4d4a7763ae00e046a163add6bbb57b5bc/sklearn/linear_model/logistic.py#L1665) in `LogisticRegressionCV.split`